actions/restore.py

```diff
@@ -227,7 +237,13 @@ def dismantle_cluster():
     for name, data in etcdctl.member_list(endpoint).items():
         if name != my_name:
             log('Disconnecting {}'.format(name), hookenv.DEBUG)
-            etcdctl.unregister(data['unit_id'], endpoint)
+            for _ in range(5):
+                try:
+                    etcdctl.unregister(data['unit_id'], endpoint)
+                    break
+                except:
```

Comment:
same as above, it's better to be specific about which exceptions are handled. This way if something unexpected happens will not be swallowed by the try catch.